src/components/VideoItem.vue

```diff
@@ -207,4 +207,12 @@ export default {
 .shorts-img {
     @apply w-full object-contain;
 }
+
+.video-info {
+    @apply mt-1 text-xs text-gray-600 font-normal;
+}
+
+.video-info .dark {
+    @apply mt-1 text-xs text-gray-300 font-normal;
+}
```

Comment:
This should be simplified to just `text-gray-300` since `.video-info` already has those classes.